interface/modules/zend_modules/module/Patientvalidation/view/layout/layout.phtml

```diff
@@ -0,0 +1,61 @@
+<?php echo $this->doctype(); ?>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+
+    <?php $this->headMeta()->appendName('viewport', 'width=device-width, initial-scale=1.0');
+    echo $this->headMeta();
+    ?>
+
+    <!-- css files -->
+    <?php foreach($this->cssFiles as $file){
+
+        $this->headLink()->appendStylesheet( $GLOBALS['assets_static_relative']  . $file);
+    }
+    echo $this->headLink();
+    ?>
+
+    <!-- Scripts -->
+    <?php
+    foreach($this->jsFiles as $file){
+
+        $this->headScript()->appendFile($GLOBALS['assets_static_relative']   . $file);
+    }
+    echo  $this->headScript() ;
+    ?>
+
+</head>
+
+<body class="container-fluid body_top " dir="<?php echo $language_direction ?>" >
```

Comment:
It's usually better to not overthink this. If it is a variable that does not contain html and it gets outputted to html, then easier to just html escape it (rather than trace where it comes from; note you never know what somebody may do with this code in the future; ie. if they decide to pull the rtl from the database etc.)